src/store/store.js

```diff
@@ -99,13 +99,25 @@ const migrations = {
       },
     };
   },
+  23: (/** @type {PersistedDefaultState} */ state) => {
```

Comment:
Why do we need this migration, whenever we deploy an update, we ask user to refresh and that will automatically update the userData object, no?

Reply:
I think you're right, but it was crashing so I added it. Maybe that wasn't the reason but let's keep it to be safe.